### PR TITLE
Code smell fixes

### PR DESCRIPTION
**Pull request by aparajit-pratap**
### Purpose
Code smell fixes.

### Declarations

Check these if you believe they are true

- [x] The codebase is in a better state after this PR
- [ ] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [ ] The level of testing this PR includes is appropriate
- [ ] User facing strings, if any, are extracted into `*.resx` files
- [ ] All tests pass using the self-service CI.
- [ ] Snapshot of UI changes, if any.
- [ ] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions) and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.

### Reviewers

@DynamoDS/dynamo

### CODE REVIEW

File: src/DynamoManipulation/DynamoManipulationExtension.cs [hunk 1]
```diff
@@ -128,14 +128,14 @@ public void Startup(ViewStartupParams viewStartupParams)
             manipulatorDaemon = ManipulatorDaemon.Create(new NodeManipulatorFactoryLoader());
         }
 
-        public void Loaded(ViewLoadedParams viewStartupParams)
+        public void Loaded(ViewLoadedParams viewLoadedParams)
```
Comment (aparajit-pratap) [inline src/DynamoManipulation/DynamoManipulationExtension.cs:131]: I'm going to change this. It hides the private field.

File: src/DynamoManipulation/DynamoManipulationExtension.cs [hunk 2]
```diff
@@ -128,14 +128,14 @@ public void Startup(ViewStartupParams viewStartupParams)
             manipulatorDaemon = ManipulatorDaemon.Create(new NodeManipulatorFactoryLoader());
         }
 
-        public void Loaded(ViewLoadedParams viewStartupParams)
+        public void Loaded(ViewLoadedParams loadedParams)
```
Comment (aparajit-pratap) [inline src/DynamoManipulation/DynamoManipulationExtension.cs:131]: I guess this won't get rid of the code smell either. Reverting ...

Comment (QilongTang) [reply src/DynamoManipulation/DynamoManipulationExtension.cs:131]: This will not.. Need to be consistent with the interface definition I believe

**Comment by reddyashish** [comment]
Looks good to me.

**Pull request merged**